rotkehlchen/constants/misc.py

```diff
@@ -14,6 +14,9 @@
 EV_DEFI = EventType('defi_event')
 EV_LEDGER_ACTION = EventType('ledger_action')
 
+CURRENCYCONVERTER_API_KEY = '7ad371210f296db27c19'
```

Comment:
This is probably okay, for the short term, but it's possible someone takes it, abuses it and gets it banned so we may end up without an api key. Having api keys in opensource is not possible. But I guess this is a free api key, so there is really no incentive to "steal it"

Also: you have an extra space below, no need for it

Reply:
Hmm that was my concern too, maybe we can provide a way to let user use their API key for this service, like the current `API Keys` page do? While we still provide a default one, since it's free

Reply:
Let's see. If this creates problems we can try that.